src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs

```diff
@@ -48,24 +48,25 @@ public class AdminIndexHandler : DatabaseRequestHandler
                     }
 
                     var index = await Database.IndexStore.CreateIndex(indexDefinition);
-                    createdIndexes.Add(index.Name);
+
+                    var indexResult = new PutIndexResult
+                    {
+                        Index = index.Name
+                    };
+                    
+                    createdIndexes.Add(indexResult);
                 }
 
                 HttpContext.Response.StatusCode = (int)HttpStatusCode.Created;
 
+                var indexesResult = new PutIndexesResult
+                {
+                    indexes = createdIndexes
+                };
+                
                 using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
                 {
-                    writer.WriteStartObject();
```

Comment:
This is a breaking change. Old clients will not work properly. You need to stay with the JSON that we had.

Reply:
used ``` context.Write(writer, new DynamicJsonArray(createdIndexes.Select(x => x.ToJson())));``` instead